src/test/java/directory_page_creator/DirectoryPageCreatorTest.java

```diff
@@ -1,48 +1,44 @@
 package directory_page_creator;
 
-import file_handler.FileHandler;
+import mocks.MockRepository;
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import repository.Repository;
+
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import java.util.List;
 
 public class DirectoryPageCreatorTest {
+    Repository repository = new MockRepository("/public");
     @BeforeEach
     public void prepFiles() {
```

Comment:
It is confusing at first to understand what exactly this class is testing. If it's just testing that the static files show up as a directory listing, then why do you have `prepFiles` and `cleanUpFiles`? Is it just because your MockRepository has a lot of stuff in it and you want your expected html result to be shorter? If so, can you think of ways to improve how your `MockRepository` class gets initialized so that it's clear in tests what the fake directory actually contains?